### PR TITLE
Migrated repo to 2022 WPILib

### LINKED ISSUE #8: Migrate code to 2022 WPILib

### PR DESCRIPTION
**Pull request by aidan-mundy**
Closes #8 

Used the WPILib 2022 auto-importer to migrate the existing code from 2021 WPILib. Merged the imported code with any relevant files that WPILib does not migrate on its own.

### CODE REVIEW

**Review by kchenturtles — APPROVED** [review]

**Comment by chit-sudeep** [comment]
Looks good

**Comment by kchenturtles** [comment]
Looks normal on read-through

**Review by chit-sudeep — COMMENTED** [review]
I scrolled through the code

Can you see my question?

**Review by chit-sudeep — APPROVED** [review]
Scrolled through the code

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,41 @@
-.gradle/
-build/
+# This gitignore has been specially created by the WPILib team.
+# If you remove items from this file, intellisense might break.
 
-.DS_Store
+### C++ ###
```
Comment (chit-sudeep) [inline .gitignore:4]: do you see the question?

**Pull request merged**